### DISCUSSION TITLE
Possible to disable flymake when shutting down eglot?

### DISCUSSION THREAD

**Discussion by doolio** [joaotavora/eglot#1379] [Q&A] [answered]
Is it possible to disable flymake (and other eglot activated minor modes) when shutting down eglot?

**Comment by joaotavora** [comment, marked as answer]
It's hard and complex to record if a mode was activated as a consequence of Eglot, and keep this record as long as its activation was motivated exclusively by Eglot.

Because of this, Eglot doesn't implement this, by default.

You can do it manually and _unconditionally_ in `eglot-managed-mode-hook`, but it will disable them even if you started them manually after or before eglot.

**Reply by doolio** [reply]
I see. Thank you for the explanation. I will try your suggestion and see if it is to my liking. Thanks again for Eglot.
